compiler-rt/lib/rtsan/rtsan_flags.inc

```diff
@@ -16,5 +16,4 @@
 // RTSAN_FLAG(Type, Name, DefaultValue, Description)
 // See COMMON_FLAG in sanitizer_flags.inc for more details.
 
-// Example flag, until we get a real one
-// RTSAN_FLAG(bool, halt_on_error, true, "If true, halt the program on error")
+RTSAN_FLAG(bool, halt_on_error, true, "Exit after first reported error.")
```

Comment:
My preference for this default is to remain true, but I'm open for discussion on the matter.

You can grep for this flag name and see the defaults for the other sanitizers, each takes their own approach